client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -266,7 +250,7 @@ class PodcastingDetails extends Component {
 						/>
 					) }
 				</FormFieldset>
-				{ podcastingFeedUrl && <PodcastFeedUrl feedUrl={ podcastingFeedUrl } /> }
+				<PodcastFeedUrl categoryId={ podcastingCategoryId } />
```

Comment:
Does it make sense to add a check for `podcastingCategoryId` or `isPodcastingEnabled` here, before rendering the component, to prevent the `connect`ed bits of `PodcastFeedUrl` from running unnecessarily?

Reply:
That makes sense to me - I just considered the `{ requiredProp && <Component /> }` pattern some sort of unspoken Calypso standard :). Whichever way you'd like to do it is good with me.

Reply:
I decided to leave this as-is... `getSelectedSiteId` is just `return state.ui.selectedSiteId` which is not expensive.  It's worth thinking about what will be called during loading/invalid states, but I think dealing with unexpected situations like this should be the responsibility of the component itself, not the caller.